### PR TITLE
Make catboost widget works in jupyter lab

### LINKED ISSUE #262: visualization in jupyter lab error
I got the following error running catboost in Jupyter Lab
"Failed to display Jupyter Widget of type CatboostIpythonWidget."

### PR DESCRIPTION
**Pull request by Dm17r1y**
Trying to fix #262 

I hereby agree to the terms of the CLA available at: [link](https://yandex.ru/legal/cla/).

### CODE REVIEW

File: catboost/python-package/mk_wheel.py [hunk 1]
```diff
@@ -229,6 +255,13 @@ def substitute_vars(file_path):
     shutil.rmtree(dir_path)
 
 
+def build_widget():
+    js_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'widget', 'js')
```
Comment (LyzhinIvan) [inline catboost/python-package/mk_wheel.py:259]: I've got `No such file error` here - building path from arc_root resolves error.

**Review by LyzhinIvan — CHANGES_REQUESTED** [review]

File: catboost/python-package/mk_wheel.py (hunk 1 above)
Comment (Dm17r1y) [reply catboost/python-package/mk_wheel.py]: Fixed: now build path from `arc_root`

**Review by LyzhinIvan — APPROVED** [review]

File: catboost/python-package/mk_wheel.py [hunk 2]
```diff
@@ -300,11 +300,15 @@ def build(arc_root, out_root, tail_args, should_build_widget):
     arc_root = os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..'))
     out_root = tempfile.mkdtemp()
 
-    widget_args = [arg for arg in sys.argv[1:] if 'BUILD_WIDGET' in arg]
-    catboost_args = [arg for arg in sys.argv[1:] if 'BUILD_WIDGET' not in arg]
+    widget_args = [arg for arg in sys.argv[1:] if arg.startswith('-DBUILD_WIDGET=')]
```
Comment (LyzhinIvan) [inline catboost/python-package/mk_wheel.py:303]: Let's make it simpler by using standard ArgumentParser:
```
widget_args_parser = ArgumentParser()
widget_args_parser.add_argument('--build-widget', choices=['yes', 'no'], default='yes')
widget_args, catboost_args = widget_args_parser.parse_known_args()
should_build_widget = widget_args.build_widget == 'yes'
```

**Review by LyzhinIvan — CHANGES_REQUESTED** [review]

**Review by LyzhinIvan — APPROVED** [review]

**Comment by LyzhinIvan** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
CLA already signed

**Comment by arcadia-devtools** [comment]
@LyzhinIvan, internal review request created: [1752441](https://a.yandex-team.ru/review/1752441)

**Comment by LyzhinIvan** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@LyzhinIvan, internal review request updated: [1752441](https://a.yandex-team.ru/review/1752441)

**Comment by cpoptic** [comment]
This may be related to https://github.com/catboost/catboost/issues/262

When fitting a CatBoost model in Visual Studio Code, with the `plot` parameter set to `True`, there are no plots shown while the model is training.
I have both `ipywidgets==7.7.0` and `catboost==1.0.5` installed and `jupyter nbextension enable --py widgetsnbextension` as per the CatBoost docs [here](https://catboost.ai/en/docs/installation/python-installation-additional-data-visualization-packages)
but still there are no plots being displayed in VS Code while fitting a Catboost model.

Why is the catboost model fitting plot not displayed while running the `.fit()` method in Visual Studio Code?  How to fix this?

**Pull request merged**